src/views/map-view.ts

```diff
@@ -290,8 +290,7 @@ export class MapView<N extends INodeBase, E extends IEdgeBase> implements IOrbVi
           this._renderer.render(this._graph);
         }
         return;
-      }
-      if (event.type === 'click' && this._strategy.onMouseClick) {
+      } else if (event.type === 'click' && this._strategy.onMouseClick) {
```

Comment:
You have `return` in the line before so you don't need `else if`. Either remove the `return` or keep `if + return` for all three cases.